src/websocket/test-websocket.c

```diff
@@ -341,7 +341,7 @@ test_parse_headers_not_enough (void)
   const gchar *input =
       "Header1: value3\r\n"
       "Header2:  field\r\n"
-      "Head3:  Anothe";
+      "Head3:  Another";
```

Comment:
@martinpitt: Not sure if this would be problematic or not.

Reply:
It's meant to simulate an incomplete/in progress transition, i. e. the first part of a data block where the stuff that follows hasn't been received yet. It's the moral equivalent of "the phone line is br...*bzzzt*" 

That said, this is harmless enough. The important point is that it does *not* have a `\r\n` terminator.

Reply:
I'm fine with this either way; it's up to you.